aplib.net-demo/Assets/Scripts/Inventory/EquipmentInventory.cs

```diff
@@ -0,0 +1,152 @@
+using Entities.Weapons;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
+/// Represents an inventory of equipment items.
+/// </summary>
+public class EquipmentInventory : MonoBehaviour
+{
+    [SerializeField] private int inventorySize = 2;
+
+    /// <summary>
+    /// Specifies whether to switch to the new item when it is equipped.
+    /// </summary>
+    /// <remarks>
+    /// If <c>true</c>, the <see cref="CurrentEquipment" /> will be automatically updated to the newly equipped item.
+    /// </remarks>
+    /// <value>
+    /// <c>true</c> if switching to the new item is enabled; otherwise, <c>false</c>.
+    /// </value>
+    /// <seealso cref="EquipmentInventory.EquipItem(IEquipment)" />
+    /// <seealso cref="EquipmentInventory.CurrentEquipment" />
+    [SerializeField] public bool SwitchToNewItem;
```

Comment:
Public fields are already serialized by default
```suggestion
    public bool SwitchToNewItem;
```